web/src/api/api.js

```diff
@@ -17,7 +17,7 @@ axiosInstance.interceptors.response.use(
     return response;
   },
   (error) => {
-    window.setApplicationError(error?.message || JSON.stringify(error), error);
+    window.setApplicationError && window.setApplicationError(error?.message || JSON.stringify(error), error);
```

Comment:
This function will always be defined except for the Jest tests. Because of this I think it's OK to do it this way.

I agree if this was a third-party library, something where the method maybe a different type, or this was being exposed as a API then it would be better to do a type check.

Reply:
That is fine, but can you add a comment there so that new comer knows why do it this way.

Reply:
Change the code to remove the 'truthy' test and instead add an empty setApplicationError method if it's not present on the window.